vignettes/load-get.Rmd

```diff
@@ -62,15 +61,16 @@ length(volumeParams)
 # The list also includes parameters of organs like "Liver|Periportal",
 # or the mucosal compartments of the intestine.
 ```
+
 Note that the path `"Organism|Kidney|*|Intracellular|Volume"` will return no parameters in the standard models, as there are no sub-containers between `Kidney` and `Intracellular`. 
 In contrast, `"Organism|Kidney|**|Intracellular|Volume"` is a valid path.
 
-The functions `getAllXXXMatching` can also be used to retrieve entities from multiple paths:
+The functions `getAllContainersMatching()`, `getAllMoleculesMatching()`, and `getAllParametersMatching()` can also be used to retrieve entities from multiple paths:
 
 ```{r getAllEntitiesMatching_multiplePaths}
-#Get the molecule Aciclovir located in `Liver|Periportal|Intracellular` and `VenousBlood|Plasma`
-molecules <- getAllMoleculesMatching(c("Organism|Liver|Periportal|Intracellular|Aciclovir",
-                                       "Organism|VenousBlood|Plasma|Aciclovir"), sim)
+# Get the molecule Aciclovir located in `Liver|Periportal|Intracellular` and `VenousBlood|Plasma`
+molecules <- getAllMoleculesMatching(c("Organism|VenousBlood|Plasma|Aciclovir",
```

Comment:
Interesting... why did you change the order of parameters?

Reply:
Because it seemed a bit odd that the second specified pattern appeared first in the output:

![image](https://user-images.githubusercontent.com/11330453/140829082-2611727a-fcfe-4fd1-8aa8-851d14efbfb9.png)

Reply:
Oh I see. Weird